src/System.Net.Primitives/tests/UnitTests/CookieInternalTest.cs

```diff
@@ -41,5 +41,67 @@ public void DomainImplicit_AddCookieWithExplicitDomain_DomainImplicitFalse()
             Assert.Equal(1, cookies.Count);
             Assert.False(cookies[CookieName].DomainImplicit);
         }
+
+        [Fact]
```

Comment:
Considering that all these tests are testing the same operations but just with different inputs, you should collapse all of these into a  singleTheory and use inlinedata attributes to pass in the various different inputs you are testing.

And in terms of the test method naming convention that we use in all our tests, we use a 3-part naming, such as:

><methodOrPropertyUnderTest>_<conditions>_<expectedResult>

So, perhaps the theory can be named "CookieParserGetString_SetCookieHeaderValue_Success" etc.